web/landing_page.go

```diff
@@ -106,10 +115,6 @@ func NewLandingPage(c LandingConfig) (*LandingPageHandler, error) {
 }
 
 func (h *LandingPageHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	if r.URL.Path != "/" {
```

Comment:
We need to keep some form of this otherwise the landing page handler will respond to `/routeprefix/somerandompage` with content. We want to 404 anything that doesn't match the RoutePrefix.